src/main/java/org/ibp/api/java/impl/middleware/dataset/validator/ObservationsTableValidator.java

```diff
@@ -83,7 +84,10 @@ private static boolean validateValue(final MeasurementVariable var, final String
 	}
 
 	private static boolean isInvalidDate(final MeasurementVariable var, final String value) {
-		return var.getDataTypeId() != null && var.getDataTypeId() == TermId.DATE_VARIABLE.getId() && !Util.isValidDate(value);
+		// Allow date in yyyy-MM-dd and yyyyDDmm format
+		final Date parsedDate = Util.tryParseDate(value, Util.FRONTEND_DATE_FORMAT);
+		final boolean isDateParseable = parsedDate != null;
+		return var.getDataTypeId() != null && var.getDataTypeId() == TermId.DATE_VARIABLE.getId() && !Util.isValidDate(value) && !isDateParseable;
 	}
```

Comment:
I wonder if it's possible to change the logic of `Util.isValidDate` instead.
Because if it's not parseable then is must be invalid too.

Reply:
Done.